webui/module/Analytics/view/analytics/analytics/runningjobs.phtml

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/**
+ *
+ * bareos-webui - Bareos Web-Frontend
+ *
+ * @link      https://github.com/bareos/bareos for the canonical source repository
+ * @copyright Copyright (c) 2013-2022 Bareos GmbH & Co. KG (http://www.bareos.org/)
+ * @license   GNU Affero General Public License (http://www.gnu.org/licenses/)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+$title = _('Analytics');
+$this->headTitle($title);
+
+?>
+
+<?php if($this->acl_alert) : echo $this->ACLAlert($this->invalid_commands); elseif(!$this->acl_alert) : ?>
+
+<ul class="nav nav-tabs">
+   <li><a href="<?php echo $this->url('analytics', array('action'=>'index')); ?>"><?php echo $this->translate('Stored Data'); ?></a></li>
+   <li class="active"><a href="<?php echo $this->url('analytics', array('action'=>'runningjobs')); ?>"><?php echo $this->translate('Running Jobs'); ?></a></li>
+</ul>
+
+<br />
+
+<div class="row">
+</div>
+
+<?php
+  echo $this->headScript()->prependFile($this->basePath() . '/js/apexcharts.min.js');
+?>
+
+<style>
+</style>
+
+<script>
+
+function getCurrentTime() {
+  let timezoneoffset = new Date().getTimezoneOffset() * 60 * 1000;
+  var currenttime = new Date().getTime();
+
+  if(timezoneoffset <= 0) {
+    currenttime += Math.abs(timezoneoffset)
+  } else {
+    currenttime -= Math.abs(timezoneoffset)
+  }
```

Comment:
```suggestion
  currenttime -= timezoneoffset
```
Not sure, why milliseconds passed since 1970-01-01 subtracted by timezone is required, but this shorter code produces the same result.

Reply:
Yes, this could be shortened. For a proper display range on the chart x-axis we need to take the timezone offset into account.

[Date.prototype.getTime()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTime)

Returns the numeric value of the specified date as the number of milliseconds since January 1, 1970, 00:00:00 UTC. (Negative values are returned for prior times.)

Reply:
Shortened.